__tests__/api.test.js

```diff
@@ -299,4 +319,42 @@ describe("GET /api/reviews where filter matches query", () => {
   });
 });
 
-describe("GET api/reviews where filter matches the last query", () => {});
+describe("GET api/reviews/:review_id/comments", () => {
+  it("should return 200: bad request when query name is incorrect", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(response => {
+        expect(response.body.comment).toEqual({
+          author: "bainesface",
+          body: "I loved this game too!",
+          comment_id: 1,
+          created_at: "2017-11-22T12:43:33.389Z",
+          review_id: 2,
+          votes: 16,
+        });
+      });
+  });
+  it("should return 400: Error when reviews/:review_id is not a number", () => {
+    return request(app)
+      .get("/api/reviews/notNumber/comments")
+      .expect(400)
+      .then(response => {
+        expect(response.body).toEqual({
+          status: 400,
+          msg: "SQL ERROR invalid user data input",
+        });
+      });
+  });
+  it("should return 404: Error when reviews/:review_id does not exist", () => {
+    return request(app)
+      .get("/api/reviews/9999/comments")
+      .expect(404)
+      .then(response => {
+        expect(response.body).toEqual({
+          status: 404,
+          msg: "This review was not found",
+        });
+      });
+  });
+});
```

Comment:
we need to test for one more situation (and alter our model):
-what if the review exists but it has no comments? in both situations your model would return an empty array, and reject the promise saying the review wasn't found.